# FTP::MakeDir

string dir_path
---
bool success
